shapely/geos.py

```diff
@@ -414,6 +414,12 @@ def errcheck_just_free(result, func, argtuple):
     else:
         return retval
 
+def errcheck_null_exception(result, func, argtuple):
+    '''Wraps errcheck_just_free, raising a TopologicalError if result is NULL'''
+    if not result:
+        raise TopologicalError("The operation '{0}' could not be performed. \
```

Comment:
Can you deal with this long string using implicit string concat instead of a forward slash?

Also I prefer docstrings to be triple double-quotes like in the examples in https://www.python.org/dev/peps/pep-0257.